src/UI/Implementation/Component/ViewControl/Renderer.php

```diff
@@ -103,6 +108,43 @@ protected function renderSectionButton(Component\Button\Button $component, $tpl,
 		$this->maybeRenderId($component, $tpl, $type."_with_id", $uptype."_PREV_ID");
 	}
 
+
+	protected function renderSortation(Component\ViewControl\Sortation $component, RendererInterface $default_renderer) {
+		$f = $this->getUIFactory();
+		$param = $component->getParameterName();
+		$options = $component->getOptions();
+		$sort_value = @$_GET[$param];
```

Comment:
I had to read up to this point to understand what the get param is used for. Change that, add a better description for Parameter name.